src/azure-cli/azure/cli/command_modules/network/tests/hybrid_2018_03_01/test_network_commands.py

```diff
@@ -51,16 +51,17 @@ def test_network_lb_zone(self, resource_group):
 
         self.kwargs.update({
             'lb': 'lb1',
+            'sku': 'standard',
             'zone': '2',
             'location': 'eastus2',
             'ip': 'pubip1'
         })
 
         # LB with public ip
-        self.cmd('network lb create -g {rg} -l {location} -n {lb} --public-ip-zone {zone} --public-ip-address {ip}')
+        self.cmd('network lb create -g {rg} -l {location} -n {lb} --sku {sku} --public-ip-zone {zone} --public-ip-address {ip}')
         # No zone on LB and its front-ip-config
         self.cmd('network lb show -g {rg} -n {lb}', checks=[
-            self.check("frontendIpConfigurations[0].zones", None),
+            self.check("frontendIPConfigurations[0].zones", None),
```

Comment:
Does such breaking changes need to be added in changelog?

Reply:
I will update the change log.